packages/flutter/test/gestures/gesture_binding_test.dart

```diff
@@ -31,7 +33,7 @@ class TestGestureFlutterBinding extends BindingBase with GestureBinding, Schedul
   }
 
   @override
-  int scheduleFrameCallback(FrameCallback callback, {bool rescheduling = false}) {
+  int addPostFrameCallback(FrameCallback callback, {bool rescheduling = false}) {
     frameCallback = callback;
```

Comment:
Change `frameCallback` to `postFrameCallback`.

Reply:
Done.